pegasus/sites.v3/code.org/public/css/brstrap.css

```diff
@@ -48,12 +48,16 @@
   .col-23 { width: 23%; float:left; }
```

Comment:
From a historical point of view these were all added for a specific case and were deliberate.  Especially on the original homepage, but also other pages, there were some pretty specific layouts that used these specific percentages.